From 3cbda5867abf6f9a9350b9726ae31897330447e4 Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@ous-research.no>
Date: Sat, 5 Aug 2023 11:08:04 +0200
Subject: [PATCH 5/8] ENH: Print external project repo variables

---
 CMake/ExternalProjectAddSource.cmake | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/CMake/ExternalProjectAddSource.cmake b/CMake/ExternalProjectAddSource.cmake
index 9fc5cecc93..362a0c6c58 100644
--- a/CMake/ExternalProjectAddSource.cmake
+++ b/CMake/ExternalProjectAddSource.cmake
@@ -233,7 +233,8 @@ macro(Slicer_Remote_Add projectname)
   #message("[${projectname}] Option (${_ep_OPTION_NAME}) value ${${_ep_OPTION_NAME}}")
   if(_add_source)
     message(STATUS "Remote - ${projectname} [OK]")
-
+    message(STATUS "Slicer_${projectname}_GIT_REPOSITORY=${_ep_GIT_REPOSITORY}")
+    message(STATUS "Slicer_${projectname}_GIT_TAG=${_ep_GIT_TAG}")
     set(_ep_args_to_pass)
     foreach(arg ${_add_source_args} ${_add_source_multi_args})
       if(_ep_${arg})
-- 
2.41.0

