From bceee10c02a2fa197ff0c7d4f87635b0df92eadc Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@ous-research.no>
Date: Thu, 4 May 2023 12:19:22 +0200
Subject: [PATCH] ENH: Add templated list of arguments for CTK

---
 SuperBuild/External_CTK.cmake | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/SuperBuild/External_CTK.cmake b/SuperBuild/External_CTK.cmake
index bfec814a4c..a75534e002 100644
--- a/SuperBuild/External_CTK.cmake
+++ b/SuperBuild/External_CTK.cmake
@@ -65,6 +65,11 @@ if(NOT DEFINED CTK_DIR AND NOT Slicer_USE_SYSTEM_${proj})
       -DCTK_QT_VERSION:STRING=5
       )
 
+    list(APPEND EXTERNAL_PROJECT_OPTIONAL_CMAKE_CACHE_ARGS
+      # Here there could be flags generated (e.g., Flatpak)
+      #<CTK_TEMPLATED_FLAGS>
+      )
+
   ExternalProject_SetIfNotDefined(
     Slicer_${proj}_GIT_REPOSITORY
     "${EP_GIT_PROTOCOL}://github.com/commontk/CTK.git"
-- 
2.39.2

