From 4da425c6635a588980ca9d19b951469eeb42ad1d Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@ous-research.no>
Date: Sun, 30 Apr 2023 22:39:08 +0200
Subject: [PATCH 1/4] ENH: Print external project variables

---
 CMake/ctkMacroCheckExternalProjectDependency.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/CMake/ctkMacroCheckExternalProjectDependency.cmake b/CMake/ctkMacroCheckExternalProjectDependency.cmake
index 872ee849..e5e26d9f 100644
--- a/CMake/ctkMacroCheckExternalProjectDependency.cmake
+++ b/CMake/ctkMacroCheckExternalProjectDependency.cmake
@@ -1114,6 +1114,7 @@ macro(ExternalProject_SetIfNotDefined var defaultvalue)
     endif()
     message(STATUS "Cache variable '${var}' set to '${_value}'")
   endif()
+message(STATUS "${var}=${${var}}")
 endmacro()
 
 #.rst:
-- 
2.41.0

