From 1723c5bc83b15e54ce6a9aee8aad3761f409cb14 Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@ous-research.no>
Date: Wed, 3 May 2023 12:21:40 +0200
Subject: [PATCH 3/4] ENH: Enable the use of offline QTTesting

---
 CMakeExternals/QtTesting.cmake | 16 +++++++++++++++-
 1 file changed, 15 insertions(+), 1 deletion(-)

diff --git a/CMakeExternals/QtTesting.cmake b/CMakeExternals/QtTesting.cmake
index e6197be6..234f4222 100644
--- a/CMakeExternals/QtTesting.cmake
+++ b/CMakeExternals/QtTesting.cmake
@@ -70,13 +70,27 @@ if(NOT DEFINED QtTesting_DIR)
         )
     endif()
   endif()
+
+  ExternalProject_SetIfNotDefined(
+    ${proj}_GIT_REPOSITORY
+    "https://github.com/commontk/QtTesting.git"
+    QUIET
+    )
+
+  ExternalProject_SetIfNotDefined(
+    ${proj}_GIT_TAG
+    b5324a213dc3c4abd7b588575ea926db57aa981e
+    QUIET
+    )
+
   message(STATUS "Adding project:${proj}")
   ExternalProject_Add(${proj}
     ${${proj}_EXTERNAL_PROJECT_ARGS}
     SOURCE_DIR ${CMAKE_BINARY_DIR}/${proj}
     BINARY_DIR ${proj}-build
     PREFIX ${proj}${ep_suffix}
-    ${location_args}
+    GIT_REPOSITORY ${${proj}_GIT_REPOSITORY}
+    GIT_TAG ${${proj}_GIT_TAG}
     CMAKE_CACHE_ARGS
       ${ep_common_cache_args}
       ${ep_cache_args}
-- 
2.39.3

