From ce0e7606b5348a8d88746c38d4d3f5023f339863 Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@ous-research.no>
Date: Wed, 3 May 2023 12:21:40 +0200
Subject: [PATCH 3/4] ENH: Enable the use of offline QTTesting

---
 CMakeExternals/QtTesting.cmake | 17 +++++++++++++++--
 1 file changed, 15 insertions(+), 2 deletions(-)

diff --git a/CMakeExternals/QtTesting.cmake b/CMakeExternals/QtTesting.cmake
index ba6b7e49..ca7a6eba 100644
--- a/CMakeExternals/QtTesting.cmake
+++ b/CMakeExternals/QtTesting.cmake
@@ -74,12 +74,25 @@ if(NOT DEFINED QtTesting_DIR)
     message(FATAL_ERROR "Support for Qt${CTK_QT_VERSION} is not implemented")
   endif()
 
-  ExternalProject_Add(${proj}
+  ExternalProject_SetIfNotDefined(
+    ${proj}_GIT_REPOSITORY
+    "https://github.com/commontk/QtTesting.git"
+    QUIET
+    )
+
+  ExternalProject_SetIfNotDefined(
+    ${proj}_GIT_TAG
+    c44b32fdea827be737e8c2f5608ffbc2e3bd08b2
+    QUIET
+    )
+
+    ExternalProject_Add(${proj}
     ${${proj}_EXTERNAL_PROJECT_ARGS}
     SOURCE_DIR ${CMAKE_BINARY_DIR}/${proj}
     BINARY_DIR ${proj}-build
     PREFIX ${proj}${ep_suffix}
-    ${location_args}
+    GIT_REPOSITORY ${${proj}_GIT_REPOSITORY}
+    GIT_TAG ${${proj}_GIT_TAG}
     CMAKE_CACHE_ARGS
       ${ep_common_cache_args}
       ${ep_cache_args}
-- 
2.41.0

