diff --git a/CMakeExternals/CTKData.cmake b/CMakeExternals/CTKData.cmake
index 53a2ff0c..68de7cae 100644
--- a/CMakeExternals/CTKData.cmake
+++ b/CMakeExternals/CTKData.cmake
@@ -29,23 +29,25 @@ if(NOT DEFINED CTKData_DIR)
     set(revision_tag ${${proj}_REVISION_TAG})
   endif()
 
-  set(location_args )
-  if(${proj}_URL)
-    set(location_args URL ${${proj}_URL})
-  elseif(${proj}_GIT_REPOSITORY)
-    set(location_args GIT_REPOSITORY ${${proj}_GIT_REPOSITORY}
-                      GIT_TAG ${revision_tag})
-  else()
-    set(location_args GIT_REPOSITORY "https://github.com/commontk/CTKData.git"
-                      GIT_TAG ${revision_tag})
-  endif()
+  ExternalProject_SetIfNotDefined(
+    ${proj}_GIT_REPOSITORY
+    "https://github.com/commontk/CTKData.git"
+    QUIET
+    )
+
+  ExternalProject_SetIfNotDefined(
+    ${proj}_GIT_TAG
+    cc07f1ff391b7828459c
+    QUIET
+    )
 
   ExternalProject_Add(${proj}
     ${${proj}_EXTERNAL_PROJECT_ARGS}
     SOURCE_DIR ${CMAKE_BINARY_DIR}/${proj}
     BINARY_DIR ${proj}-build
     PREFIX ${proj}${ep_suffix}
-    ${location_args}
+    GIT_REPOSITORY ${${proj}_GIT_REPOSITORY}
+    GIT_TAG ${${proj}_GIT_TAG}
     CONFIGURE_COMMAND ""
     BUILD_COMMAND ""
     INSTALL_COMMAND ""
diff --git a/CMakeExternals/QtTesting.cmake b/CMakeExternals/QtTesting.cmake
index e6197be6..234f4222 100644
--- a/CMakeExternals/QtTesting.cmake
+++ b/CMakeExternals/QtTesting.cmake
@@ -70,13 +70,27 @@ if(NOT DEFINED QtTesting_DIR)
         )
     endif()
   endif()
+
+  ExternalProject_SetIfNotDefined(
+    ${proj}_GIT_REPOSITORY
+    "https://github.com/commontk/QtTesting.git"
+    QUIET
+    )
+
+  ExternalProject_SetIfNotDefined(
+    ${proj}_GIT_TAG
+    b5324a213dc3c4abd7b588575ea926db57aa981e
+    QUIET
+    )
+
   message(STATUS "Adding project:${proj}")
   ExternalProject_Add(${proj}
     ${${proj}_EXTERNAL_PROJECT_ARGS}
     SOURCE_DIR ${CMAKE_BINARY_DIR}/${proj}
     BINARY_DIR ${proj}-build
     PREFIX ${proj}${ep_suffix}
-    ${location_args}
+    GIT_REPOSITORY ${${proj}_GIT_REPOSITORY}
+    GIT_TAG ${${proj}_GIT_TAG}
     CMAKE_CACHE_ARGS
       ${ep_common_cache_args}
       ${ep_cache_args}
